rocketpool-cli/node/sync.go

```diff
@@ -50,7 +50,7 @@ func printSyncProgress(status *api.ClientManagerStatus, name string) {
 func getSyncProgress(c *cli.Context) error {
 
 	// Get RP client
-	rp, err := rocketpool.NewClientFromCtx(c)
```

Comment:
this command cares whether the clients are synced, but handles it further down, of course.